### PR TITLE
Pull request lab3

### PR DESCRIPTION
**Pull request by ali-clara**
@jarods00

### CODE REVIEW

File: Lab_3.ipynb [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00969551-c783-46e0-8e04-a656ed236e1b",
+   "metadata": {},
+   "source": [
+    "Statistical question: How many data points lie outside a chosen threshold of a Gaussian distribution with mean 12 and width 0.4?"
```
Comment (jarods00) [inline Lab_3.ipynb]: Why is this the statistical question you chose?

File: Lab_3.ipynb [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f92718ee-b11d-4af1-976c-326ee368f96f",
+   "metadata": {},
+   "source": [
+    "$N_{outside} = \\int_{-\\infty}^{-5\\sigma}T(x)dx +  \\int_{{5\\sigma}}^{\\infty}T(x) dx  $, where $T(x)$ is the temperature frequency distribution. We have chosen a statistical threshold of 5$\\sigma$."
```
Comment (jarods00) [inline Lab_3.ipynb]: Why a 5 sigma threshold?

File: Lab_3.ipynb [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+    "df = pd.DataFrame(data=my_data)\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f293299a-4900-4120-b8c9-d64b7772139b",
+   "metadata": {},
+   "source": [
+    "**B)** There were no omissions of good data with the chosen statistical threshold. If the threshold were to tighten, however, it would begin to remove more and more good data."
```
Comment (jarods00) [inline Lab_3.ipynb:206]: Is there a way to determine exactly how this might depend on the threshold?

File: Lab_3.ipynb [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+   "cell_type": "markdown",
+   "id": "7b9bf64e-8d3d-4d07-92ce-40147b465cd7",
+   "metadata": {},
+   "source": [
+    "**Statistical Question:** Assuming a null-hypothesis background, what signal (distance travelled) is associated with a 5$\\sigma$ significance? \n",
+    "\n",
+    "Distance is given by $\\sqrt{X^2 + Y^2}$. Both X and Y are independent Gaussian distribtions, and the Euclidian norm of two Gaussians produces a Rayleigh. Therefore, the integral is\n",
+    "\n",
+    "\\begin{aligned}\n",
+    "Probability = \\int_{{d}}^{\\infty}pdf(x) dx\n",
```
Comment (jarods00) [inline Lab_3.ipynb:268]: Isn't probability here the probability associated with the 5 sigma value, not the actual 5 sigma value itself?

**Comment by ali-clara** [comment]
@hrzhao76

**Pull request merged**